ci/prometheus-down.sh

```diff
@@ -7,53 +7,49 @@
 # 
 # If the query fails, then something is wrong with Prometheus's query API.
 # 
-# If at least one timestamp is up to date, then we know it's probably working,
+# If at tsdb max timestamp is up to date, then we know it's probably working,
 # otherwise, something is jammed up inside probably.
-# 
-# 
-# XXX If something stops the processes that generate these timestamp metrics,
-# this will result in false positives.  Maybe a deploy might do this?
+#
 #
 
 set -u
 
 : ${PROMETHEUSHOST:="0.prometheus.production-monitoring.prometheus-production.toolingbosh"}
 : ${ALERTMANAGERHOST:="0.alertmanager.production-monitoring.prometheus-production.toolingbosh"}
-: ${QUERIES:="
-  push_time_seconds
-"}
+: ${QUERY:="prometheus_tsdb_head_max_time%7Binstance%3D\"localhost:9090\",job%3D\"prometheus\"%7D"}
```

Comment:
Looks like this query gives us milliseconds, so we might need to scale for the subtraction to work.

Reply:
Good catch.